main/cmake/APKBUILD

```diff
@@ -41,6 +41,7 @@ build() {
 		--system-curl \
 		--system-expat \
 		--no-system-jsoncpp \
+		--no-system-librhash \
```

Comment:
Oops, sorry.... That is outdated. Jsoncpp uses scons now.

https://git.alpinelinux.org/cgit/aports/tree/main/jsoncpp/APKBUILD